packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -437,6 +437,34 @@ class ToolbarOptions {
 ///  * When the virtual keyboard pops up.
 /// {@endtemplate}
 ///
+/// {@template flutter.widgets.editableText.accessibility}
+/// ## Troubleshooting Common Accessibility Issues
+///
+/// ### Customizing User Input Accessibility Announcements
+///
+/// To customize user input accessibility announcements triggered by text
+/// changes, use [SemanticsService.announce] to make the desired
+/// accessibility announcement.
+///
+/// {@tool snippet}
+/// On iOS, the on-screen keyboard may announce the most recent input
```

Comment:
does it work on Android?

Reply:
`SemanticsService.announce` should but I'm not sure the same problem exists on Android (the internal issue report didn't mention anything about Android so I assume it works fine even without the workaround on Android).